modules/cats/shared/src/main/scala/eu/timepit/refined/cats/CatsValidated.scala

```diff
@@ -0,0 +1,11 @@
+package eu.timepit.refined.cats
+
+import cats.data.{NonEmptyList, Validated, ValidatedNel}
+import eu.timepit.refined.api.RefinedTypeOps
+
+object validation {
+  implicit class CatsValidateOps[FTP, T](companion: RefinedTypeOps[FTP, T]) {
+    def validate(value: T): ValidatedNel[String, FTP] =
+      Validated.fromEither(companion.from(value)).leftMap(NonEmptyList.one)
```

Comment:
There is a `toValidatedNel` on Cats' `EitherSyntax` which would allow us to just write `companion.from(value).toValidatedNel` here.